django/core/management/base.py

```diff
@@ -71,6 +72,15 @@ def error(self, message):
         else:
             raise CommandError("Error: %s" % message)
 
+    def add_subparsers(self, **kwargs):
+        parser_class = kwargs.get("parser_class", type(self))
+        if issubclass(parser_class, CommandParser):
+            kwargs["parser_class"] = partial(
+                parser_class,
+                called_from_command_line=self.called_from_command_line,
```

Comment:
Hmm. Should we also pass `missing_args_message` to the subparser?

Reply:
I just delved into the guts of this a little. Testing in Python 3.8.16, it appears that `command_parser.parse_args()` is only ever called on the main parser and not on the subparsers, so `missing_args_message` is never used by a subparser.

Even if you overcame that issue, each subparser would need a `missing_args_message` specific to it and its required arguments so it doesn't make sense to pass the main parsers value through as a default.

Reply:
Cool. Thanks for checking!